File: reactor/r2dbc-mysql/src/main/java/ua/mai/zyme/r2dbcmysql/entity/Transfer.java
package ua.mai.zyme.r2dbcmysql.entity;

import lombok.Builder;
import lombok.Data;
import org.springframework.data.annotation.*;
import org.springframework.data.relational.core.mapping.Column;

import java.time.LocalDateTime;

@Builder
@Data
public class Transfer {

  @Id
  @Column("transfer_id")
  private Long transferId;

  @Column("from_member_id")
  private Integer fromMemberId;

  @Column("to_member_id")
  private Integer toMemberId;

  private Long amount;

  @Column("created_date")
  private LocalDateTime createdDate;

}

File: kotlin/kotlin-r2dbc-mysql/kotlin-r2dbc-mysql-be/src/main/java/ua/mai/zyme/r2dbcmysql/entity/Transfer.kt
package ua.mai.zyme.r2dbcmysql.entity

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import java.time.LocalDateTime

data class Transfer @JvmOverloads constructor(
    @Id
    @Column("transfer_id")
    var transferId: Long? = null,

    @Column("from_member_id")
    var fromMemberId: Int? = null,

    @Column("to_member_id")
    var toMemberId: Int? = null,

    var amount: Long? = null,

    @Column("created_date")
    var createdDate: LocalDateTime? = null
)